spring-cloud-scheduler-spi/src/main/java/org/springframework/cloud/scheduler/spi/implementation/DuplicateScheduleException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.scheduler.spi.implementation;
+
+/**
+ * Thrown when a schedule is already exists when scheduler the gets a request
+ * to deploy it.
+ *
+ * @author Glenn Renfro
+ */
+public class DuplicateScheduleException extends RuntimeException{
```

Comment:
Not clear to me which method in `Scheduler` would throw this.  `CreateScheduleException` and `UnScheduleException` pair with the methods `schedule` and `unschedule`.